t/t3428-rebase-edit-todo.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+test_description='rebase should reread the todo file if an exec modifies it'
+
+. ./test-lib.sh
+
+todo=$(git rev-parse --git-dir)/rebase-merge/git-rebase-todo
```

Comment:
I wonder why you do not need double quotes here, despite the fact that we are in a directory with spaces in its name. I guess it is because `--git-dir` returns `.git` here.